docker/localsettings.py

```diff
@@ -145,16 +145,6 @@
 
 BITLY_OAUTH_TOKEN = None
 
-####### Jar signing config ########
-
-_ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
-JAR_SIGN = {
```

Comment:
So great to finally get rid of these. I think I've tried removing them at least twice before 😆